### ISSUE TITLE
Missing line numbers in timeslider

### ISSUE THREAD

**Issue by webzwo0i** [ether/etherpad#4669]
I think this worked once, maybe it was removed intentionally, but I thought it is useful

On
https://video.etherpad.com/p/list-numbering/timeslider#165 
the line numbers are not shown

**Comment by JohnMcLear** [comment]
yeah that's weird, I don't recall dropping it.  Will investigate.

**Comment by JohnMcLear** [comment]
https://www.npmjs.com/package/ep_timesliderdiff is worth referring to here.  Note that timeslider never had line numbers, not even back in the day.. 

So is this a bug?  I don't think so, but I would say that your line numbers should be present in the time-slider if you have them present in the editor.  

It's technically not a bug tho as it's expected behaviour at this point albeit not ideal 🖌️
